presto-mysql/pom.xml

```diff
@@ -157,5 +157,13 @@
             <artifactId>annotations</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
why this is needed?

Reply:
It's for `StringUtils.rightPad` used in `testDecimalExceedingPrecisionMaxWithDecimalMapping` to create the expected string decimal values when failing to `CONVERT_TO_VARCHAR` option.

Reply:
I think the best practice is to use hardcoded expected values so reader may understand the test easier. I would remove this dependency.